tests/wallet/did_wallet/test_did_rpc.py

```diff
@@ -1,123 +1,57 @@
-import logging
+import asyncio
 
 import pytest
 
-from chia.rpc.rpc_server import start_rpc_server
+from chia.consensus.block_rewards import calculate_base_farmer_reward, calculate_pool_reward
 from chia.rpc.wallet_rpc_api import WalletRpcApi
-from chia.rpc.wallet_rpc_client import WalletRpcClient
 from chia.simulator.simulator_protocol import FarmNewBlockProtocol
 from chia.types.peer_info import PeerInfo
-from chia.util.ints import uint16, uint64
-from chia.wallet.did_wallet.did_wallet import DIDWallet
-from chia.wallet.util.wallet_types import WalletType
+from chia.util.ints import uint16, uint32
 from tests.time_out_assert import time_out_assert
-from tests.util.socket import find_available_listen_port
 
-log = logging.getLogger(__name__)
 
-pytestmark = pytest.mark.skip("TODO: Fix tests")
-
-
-class TestDIDWallet:
+class TestDIDRPC:
     @pytest.mark.asyncio
-    async def test_create_did(self, bt, three_wallet_nodes, self_hostname):
-        num_blocks = 4
-        full_nodes, wallets = three_wallet_nodes
+    async def test_did_get_set_wallet_name(self, wallet_node_simulator):
```

Comment:
It seems like we are not testing the other DID RPC methods. Every method should be tested here

Reply:
oh NVM it looks like they are tested in `test_wallet_rpc`. is there a reason for dividing the tests in two?